config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root 'groups#index'
+
+  resources :groups, only: [:index, :show, :new, :create] do
+    resources :entities, only: [:index]
+  end
+
+  resources :entities, only: [:new, :create]
```

Comment:
- [ ] Please I think there's an issue here. It make it very clear int he Entity model that an entity belongs to a group. But you just isolate out some actions which I think you should fix it. This URL `http://localhost:3000/groups/1` display all entities(index) instead of http://localhost:3000/groups/entities. Entities belong to one or several groups

Reply:
Done :ok_hand: